### PR TITLE
Various fixes for Ginkgo release

### PR DESCRIPTION
**Pull request by pomegranited**
Many fixes noticed during a Gingko Insights/pipeline deployment.

Please see commit comments for details on each change.

**Reviewer**
- [x] @UmanShahzad

### CODE REVIEW

File: docs/analytics/resources/vars-analytics.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ EDXAPP_LMS_BASE_SCHEME: 'x'
 EDXAPP_LMS_BASE: 'x'   # in vars it might be set to {{ LMS_SITE_NAME }} - in such case either copy LMS_SITE_NAME here
                        # or just copy value. The idea is to have EDXAPP_LMS_BASE set to the same value here and in
                        # vars.yml
+EDXAPP_CMS_BASE: 'x'
```
Comment (UmanShahzad) [inline docs/analytics/resources/vars-analytics.yml:7]: Holy moly, so that's why...

File: docs/analytics/resources/vars-analytics.yml [hunk 2]
```diff
@@ -8,6 +8,9 @@ EDXAPP_CMS_BASE: 'x'
 EDXAPP_CONTACT_EMAIL: 'x'
 COMMON_MYSQL_MIGRATE_USER: 'x'
 COMMON_MYSQL_MIGRATE_PASS: 'x'
+
+OPENEDX_RELEASE: 'master' # For stability, choose a specific release, e.g. open-release/ficus.3
```
Comment (UmanShahzad) [inline docs/analytics/resources/vars-analytics.yml:12]: Thank you :)

File: docs/analytics/resources/vars-analytics.yml (hunk 1 above)
Comment (pomegranited) [reply docs/analytics/resources/vars-analytics.yml:7]: IKR?

**Review by UmanShahzad — COMMENTED** [review]
Thanks @pomegranited ! I have not explicitly gone ahead and tested these changes, but will have confidence that it's good when testing the most recent insights deployment you're going to make. Will then give :+1: Other than that, these are great changes :)

**Review by UmanShahzad — APPROVED** [review]
:+1:

- [x] I tested this: I read through the doc to make sure everything's sensibly worded, grammatically correct, and has no markdown-related issues. I trust that the steps are appropriate because they brought up a working Insights instance.
- [x] I read through the code
- [x] I checked for accessibility issues
- [x] Includes documentation

**Pull request merged**